packages/core/content-type-builder/admin/src/components/ContentTypeBuilderNav/tests/index.test.js

```diff
@@ -9,9 +9,9 @@ import { render } from '@testing-library/react';
 import { createMemoryHistory } from 'history';
 import React from 'react';
 import { Router } from 'react-router-dom';
-import LanguageProvider from '@strapi/admin/admin/src/components/LanguageProvider/LanguageProvider';
-import Theme from '@strapi/admin/admin/src/components/Theme/Theme';
-import ThemeToggleProvider from '@strapi/admin/admin/src/components/ThemeToggleProvider/ThemeToggleProvider';
+import LanguageProvider from '@strapi/admin/admin/src/components/LanguageProvider';
```

Comment:
Do we need to add `@strapi/admin` as depdency to the CTB then?

Reply:
same It didn't complain will verify

Reply:
Ok then 🔥